tests/acceptance/features/bootstrap/SettingsContext.php

```diff
@@ -405,12 +402,13 @@ public function sendRequestToSwitchSystemLanguage(string $user, string $language
 	 * @return void
 	 *
 	 * @throws Exception
+	 * @throws GuzzleException
 	 */
-	public function theUserHasSwitchedSysemLanguage(string $user, string $language): void {
-		$this->sendRequestToSwitchSystemLanguage($user, $language);
+	public function theUserHasSwitchedSystemLanguage(string $user, string $language): void {
 		$this->featureContext->theHTTPStatusCodeShouldBe(
```

Comment:
please, do like this.
```suggestion
                $response = $this->sendRequestToSwitchSystemLanguage($user, $language);
		$this->featureContext->theHTTPStatusCodeShouldBe(
		        "Expected response status code should be 201",
			$response
		);
```